### PR TITLE
Adding plotAbundanceDensity example.

### PR DESCRIPTION
**Pull request by ChouaibB**
Hi,

Hereby an example of `plotabundanceDensity` from `miaViz` is added to the [data exploration](https://microbiome.github.io/OMA/microbiome-exploration.html) chapter of OMA.

### CODE REVIEW

File: 13_microbiome_exploration.Rmd [hunk 1]
```diff
@@ -159,6 +159,34 @@ plotRowTree(x[rowData(x)$Phylum %in% top_phyla_mean,],
...
+
+In the following few demonstrations are shown, using the 
+[Lahti L et al. (2014)](https://doi.org/10.1038/ncomms5344) dataset.
+
+A Jitter plot using the top 25 taxa as a default value could be produced as follows: 
+
+```{r, warning=FALSE, message=FALSE}
+library(microbiomeDataSets)
+tse <- LahtiWAData()
+plotAbundanceDensity(tse, abund_values = "counts")
```
Comment (antagomir) [inline 13_microbiome_exploration.Rmd:176]: Can you replace this one with 

plotAbundanceDensity(tse, layout = "jitter", abund_values = "relabundance", n = 40, point_size=1, point_shape=19, point_alpha=0.1) + scale_x_log10(label=scales::percent)

and then related to this also add citation to Fig S1 in [Salosensaari et al. 2021](https://www.nature.com/articles/s41467-021-22962-y) where we used a similar illustration.

**Review by antagomir — COMMENTED** [review]
Seems good.

File: 13_microbiome_exploration.Rmd (hunk 1 above)
Comment (ChouaibB) [reply 13_microbiome_exploration.Rmd:176]: Sure, there seems btw to be some problem at the check related to installing dependencies.

**Review by antagomir — CHANGES_REQUESTED** [review]
Remaining changes seem minor. Can we complete & close this?

File: book.bib [hunk 2]
```diff
@@ -519,3 +519,19 @@ @article{Chao1984
...
+
+@article{LahtiL2014,
+    author = {Lahti L et al.},
+    title = {Tipping elements in the human intestinal ecosystem},
+    journal = {Nature Communications},
+    doi = {https://doi.org/10.1038/ncomms5344},
+    year = {2014}
+}
+
+@article{SaloS2021,
```
Comment (antagomir) [inline book.bib:531]: Use the full author surname for references "Salosensaari2021" so we are consistent. Makes managing easier,

File: book.bib [hunk 3]
```diff
@@ -519,3 +519,19 @@ @article{Chao1984
     url = {https://www.jstor.org/stable/4615964},
     year = {1984}
 }
+
+@article{LahtiL2014,
```
Comment (antagomir) [inline book.bib:523]: Just Lahti2014

File: book.bib [hunk 4]
```diff
@@ -519,3 +519,19 @@ @article{Chao1984
...
+    doi = {https://doi.org/10.1038/ncomms5344},
+    year = {2014}
+}
+
+@article{SaloS2021,
+    author = {Salosensaari et al.},
+    title = {Taxonomic signatures of cause-specific mortality risk in human gut microbiome},
+    journal = {Nature Communications},
+    url = {https://www.nature.com/articles/s41467-021-22962-y},
+    year = {2021}
```
Comment (antagomir) [inline book.bib:536]: Must include vol, issue, and page info

File: book.bib [hunk 5]
```diff
@@ -519,3 +519,19 @@ @article{Chao1984
...
     year = {1984}
 }
+
+@article{LahtiL2014,
+    author = {Lahti L et al.},
+    title = {Tipping elements in the human intestinal ecosystem},
+    journal = {Nature Communications},
+    doi = {https://doi.org/10.1038/ncomms5344},
+    year = {2014}
+}
```
Comment (antagomir) [inline book.bib:529]: include vol, issue and page info

File: 13_microbiome_exploration.Rmd [hunk 6]
```diff
@@ -166,11 +166,10 @@ the function `plotAbundanceDensity` where most abundant taxa can be plotted
 including several options.
 
 In the following few demonstrations are shown, using the 
-[Lahti L et al. (2014)](https://doi.org/10.1038/ncomms5344) dataset.
+[@LahtiL2014] dataset.
 
 A Jitter plot based on relative abundance data, similar to the one presented at 
-[Salosensaari et al. 2021](https://www.nature.com/articles/s41467-021-22962-y) 
-supplementary figure 1, could be visualized as follows: 
+[@SaloS2021] supplementary figure 1, could be visualized as follows:
```
Comment (antagomir) [inline 13_microbiome_exploration.Rmd:172]: See reference comments

File: 13_microbiome_exploration.Rmd [hunk 7]
```diff
@@ -166,11 +166,10 @@ the function `plotAbundanceDensity` where most abundant taxa can be plotted
 including several options.
 
 In the following few demonstrations are shown, using the 
-[Lahti L et al. (2014)](https://doi.org/10.1038/ncomms5344) dataset.
+[@LahtiL2014] dataset.
```
Comment (antagomir) [inline 13_microbiome_exploration.Rmd:169]: See reference comments

**Pull request merged**